frontend/public/kubevirt/components/modals/create-vm-modal.js

```diff
@@ -21,8 +21,8 @@ export const openCreateVmWizard = ( activeNamespace, createTemplate = false ) =>
       resource: getResource(NamespaceModel),
     },
     virtualMachines: {
-      resource: getResource(VirtualMachineModel),
-      required: true,
+      resource: getResource(VirtualMachineModel, {namespace: activeNamespace}),
+      requiredIfAuthorized: true,
```

Comment:
can we not add another feature to `withResources` ? It will be harder to migrate to upstream openshift. Adding `{namespace: activeNamespace}` is enough IMHO and is consistent with what we do with other resources.

Reply:
This will not influence that so much. The VM dialog and the resource passing have to be  implemented by either wrapper component or by modifying VM dialog. Either way it will not be a problem to include this functionality there.

I think it is better for a dialog to depend on less things than it needs to. The existing VM error will be resolved at the end of the dialog instead of at the beginning if VMs are not present.

Reply:
ok, makes sense :)